### DISCUSSION TITLE
8월 10일 Git Actions bugfix

### DISCUSSION THREAD

**Discussion by silverpoodle** [cca-ffodregamdi/running-hi-back#34] [Trouble Shooting]
## 1. Git Action Secret Key 설정

```java
      - name: Setup MySQL
        uses: samin/mysql-action@v1
        with:
          mysql database: 'runninghi'
          mysql user: ${{ secrets.MYSQL_USERNAME }}
          mysql password: ${{ secrets.MYSQL_PASSWORD }}
```

<img width="638" alt="image" src="https://github.com/cca-ffodregamdi/running-hi-back/assets/88484476/ea369de9-85f3-4f9a-beae-452419d41e51">

DB에 접근하기 위한 user이름과 비밀번호를 따로 관리

## 2. Git Test 결과 출력

```java
      - name: Publish Unit Test Results
        uses: EnricoMi/publish-unit-test-result-action@v2
        if: always()
        with:
          junit_files: build/test-results/test/**/*.xml
```
<img width="861" alt="스크린샷 2023-08-10 193952" src="https://github.com/cca-ffodregamdi/running-hi-back/assets/88484476/e7286cf6-b8d3-4d27-8654-1cb39570a03d">

## 3. 주의 할 점
 - data.sql 작성 시 테이블 명은 소문자로!
 ```sql
INSERT INTO tbl_bookmark(user_no, folder_no, post_no)
VALUES(1,1,1);
```

- build.gradle 의존성 추가
```
implementation 'mysql:mysql-connector-java:8.0.25'
```

**Comment by Subak-Uncle** [comment]
어려운 문제 해결해주셨네요!! 감사합니다:)
